ui/apps/platform/src/Containers/MainPage/AuthenticatedRoutes.tsx

```diff
@@ -29,11 +29,10 @@ function AuthenticatedRoutes(): ReactElement {
         case AUTH_STATUS.AUTH_PROVIDERS_LOADING_ERROR:
         case AUTH_STATUS.LOGIN_AUTH_PROVIDERS_LOADING_ERROR:
             return (
-                <Redirect
-                    to={{
-                        pathname: '/login',
-                        state: { from: `${location.pathname}${location.search}` },
-                    }}
+                <Navigate
+                    to="/login"
+                    state={{ from: `${location.pathname}${location.search}` }}
```

Comment:
Aha, is this where the path comes from? (pardon pun)

What do you think about a comment here? For our future selves :)